src/workerd/server/workerd-api.c++

```diff
@@ -56,6 +56,7 @@ namespace workerd::server {
 
 using api::pyodide::PythonConfig;
 
+namespace {
```

Comment:
Adding this closure is high-impact because JsgWorkerdIsolate instantiates the entire workerd JS API – on macOS compiling without optimization sees little impact to compile time or object code size (41.1MB => 40.6MB), but under -O3 there is a large impact on both (38.2s => 51.8s, 15.1MB => 10.3MB). Compiling workerd-api.c++ is a bottleneck for the build, but I think this is worth it since such tradeoffs are what we are looking for when optimization is enabled, we might also see faster LTO link times as there is less code to be optimized at that stage.